### PR TITLE
Adds Columbine Kit (RELIES ON PR #6535)

### PR DESCRIPTION
**Pull request by asskek**
Adds the Columbine kit, as seen here https://hippiestation.com/threads/columbine-kit.7429/.  Requires Spiked Baseball Bats to be merged. They were originally in one PR but Tortellini "Tourettes" Tony and TDFoolguy sperged about it, so here it is, separate. 

RELIES ON PR #6545

### CODE REVIEW

**Comment by tortellinitony** [comment]
build this

File: hippiestation/code/game/objects/items/storage/uplink_kits.dm [hunk 1]
```diff
@@ -69,3 +69,45 @@
...
+		new /obj/item/clothing/suit/jacket/leather/overcoat(src)
+		new /obj/item/clothing/mask/bandana/black(src)
+		new /obj/item/storage/belt/bandolier(src)
+		new /obj/item/clothing/shoes/combat(src)
+		new /obj/item/clothing/gloves/fingerless(src)
+	var/ericBag = new /obj/item/storage/backpack/duffelbag(src)
+	new /obj/item/switchblade(ericBag)
+	new /obj/item/melee/baton/cattleprod(ericBag)
+	new /obj/item/grenade/plastic/x4(ericBag)
+	new /obj/item/melee/baton/cattleprod(ericBag)
```
Comment (GuyonBroadway) [inline hippiestation/code/game/objects/items/storage/uplink_kits.dm]: might wanna make this obj/item/melee/baton/cattleprod/hippie_cattleprod else you will get the shitty /tg/ cattleprod that doesn't fit in a bag.

Comment (asskek) [reply hippiestation/code/game/objects/items/storage/uplink_kits.dm]: Alright, didn't realize that, thank you!

**Comment by JohnGinnane** [comment]
also @asskek pls)

**Comment by ghost** [comment]
+1

**Comment by Pyko1** [comment]
+1

**Comment by ghost** [comment]
IDK maybe I'm being lame but I don't like the idea of it being called "columbine kit"

**Comment by praisenarsie** [comment]
stale, ping for reopen

**Pull request closed without merging**